tests/framework/results/mocks.py

```diff
@@ -76,3 +76,14 @@ def verify_stratification_added(
             matching_stratification_found = True
             break
     return matching_stratification_found
+
+
+# Mock for get_value call for Pipelines, returns a str instead of a Pipeline
+def mock_get_value(self, name: str):
+    try:
+        name.__hash__()
```

Comment:
Is it better to try to hash it here or just to check if it's a string?

Reply:
The error I'm trying to handle is the TypeError unhashable, but I might as well add another check if it's a string and raise TypeError if not.